pkg/storaged/storage-anaconda.scss

```diff
@@ -19,3 +19,15 @@
     --pf-v6-c-card--BorderColor: transparent;
   }
 }
+
+body:has(.storage-anaconda) {
```

Comment:
today we merged a PR which renamed the class from storage-anaconda to anaconda. Please rebase and adjust.